HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/APIResource.swift

```diff
@@ -114,28 +120,28 @@ struct APIResource<T: Decodable>: Resource {
         case .createDocument(_, _, let mimeSubType, let documentType):
```

Comment:
[suggestion]: What do you say to improve this property to Reuse common Accept and Content-Type header values instead of repeating them across cases?
here is my suggestion to replace the entire ` var defaultHeaders: HTTPHeaders {..}` with:

```
var defaultHeaders: HTTPHeaders {
        // Define common headers
        let acceptHeader = ["Accept": ContentType.content(
            version: apiVersion,
            subtype: nil,
            mimeSubtype: MimeSubtype.json.rawValue
        ).value]

        // Helper method to construct Content-Type header
        func contentTypeHeader(subtype: String?, mimeSubtype: String) -> HTTPHeaders {
            return ["Content-Type": ContentType.content(
                version: apiVersion,
                subtype: subtype,
                mimeSubtype: mimeSubtype
            ).value]
        }

        switch method {
            case .createDocument(_, _, let mimeSubType, let documentType):
                return acceptHeader.merging(contentTypeHeader(
                    subtype: documentType?.name,
                    mimeSubtype: mimeSubType.rawValue
                )) { _, new in new }

            case .file(_):
                return [:]

            case .paymentProviders, .paymentProvider(_), .paymentRequests(_, _):
                return acceptHeader

            case .pdfWithQRCode(_, let mimeSubtype):
                return ["Accept": ContentType.content(
                    version: apiVersion,
                    subtype: nil,
                    mimeSubtype: mimeSubtype.rawValue
                ).value]

            default:
                // Default headers for other cases
                return acceptHeader.merging(contentTypeHeader(
                    subtype: nil,
                    mimeSubtype: MimeSubtype.json.rawValue
                )) { _, new in new }
        }
    }
```